readme/sdkDevelopment.md

```
@@ -0,0 +1,69 @@
+2. Navigate to the project directory.
+3. Run `yarn install` to install all dependencies.
+
+### Official Documentation
+
+Explore the full documentation and access example use cases by visiting the [VeChain SDK Documentation](https://docs.vechain.org/developer-resources/sdks-and-providers/sdk)
+
+### Commands
+
+- **Build**: Execute `yarn build` to build the project.
+- **Test**: Execute `yarn test:solo` to run all tests.
+  - **NOTE**: Integration tests require a local thor-solo node. See the [Integration Testing](#integration-testing) section for more details. 
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)